Core/Src/main.c

```diff
@@ -65,6 +68,13 @@ void SystemClock_Config(void);
 int main(void)
 {
   /* USER CODE BEGIN 1 */
+	uint8_t spi_rx_buf[3]; //receive buffer
+	uint8_t spi_tx_buf[3]; //transmit buffer
```

Comment:
How many bytes are transmitted and received in one round of communication with the ADC? I believe it is in the ADC datasheet section 5 and or 6.

Reply:
It is 3, ignore me, I cannot read a 3 from a 2.